api/dummy.go

```diff
@@ -0,0 +1,29 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/ONSdigital/log.go/v2/log"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+)
+
+/*
+   dummyHandler is used to log a Filter Flex
+   request that does not have an equivalent
+   CMD Journey.
+
+   Used currently for DELETE filters/{id}/dimensions/{name}/options
+*/
+func (api *FilterAPI) dummyHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	filterBlueprintID := vars["filter_blueprint_id"]
+
+	logData := log.Data{
+		"filter_blueprint_id": filterBlueprintID,
+	}
+
+	ctx := r.Context()
+	log.Error(ctx, "Should not hit this endpoint", errors.New("unhandled state"), logData)
```

Comment:
No blank line at the end of file.. Also it does actually need to handle a request if it does end up here by accident. There is an example of another endpoit The logged error message is also bad, it won't make any sense when seen in the logs (there's no mention of what endpoint it is, for one). Also you should keep the language more formal ('invalid request destination' or something). Look at `api.postFilterBlueprintSubmitHandler` for a good example.

Reply:
The endpoint now returns a bad request, with logging of the route taken, and more formal language. I will leave you to resolve